x/src/test.rs

```diff
@@ -100,6 +146,38 @@ pub fn run(mut args: Args, config: Config) -> Result<()> {
             all_features,
             ..base_args
         })?;
-        Ok(())
     }
+
+    if args.html_cov_dir.is_some() {
+        let mut debug_dir = project_root();
+        debug_dir.push("target/debug/");
```

Comment:
Looks like there is a semantic conflict with one of Rain's PRs that changed the return type of `project_root`. Change this to use [`join`](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.join) and it should work

Reply:
thanks, looking.   For now, this is still generating the correct dir.    "grcov" **"/Users/rexhoffman/git/libra/target/debug/"** "-s" "/Users/rexhoffman/git/libra" "-t" "html" "--llvm" "--branch" "--ignore" "/*" "--ignore" "x/*" "--ignore" "testsuite/*" "--ignore-not-existing" "-o" "/Users/rexhoffman/test2".

Not bothering with the the nonnested crate testing right now (ran in common/lcs), as without extending --no-fail-fast consensus dies all the time anyway.

Reply:
scratch that.  wan't on latest upstream.... sigh.